app/src/main/java/atlant/moviesapp/presenters/MovieListPresenter.java

```diff
@@ -82,6 +93,59 @@ public void onFailure(Call<MoviesResponse> call, Throwable t) {
 
     }
 
+    public void postFavorite(int id, String session_id, BodyFavourite favorite) {
+        final ApiInterface apiservice = ApiClient.getClient().create(ApiInterface.class);
+        postCall = apiservice.addFavorite(id, API_KEY, session_id, favorite);
+        postCall.enqueue(new Callback<PostResponse>() {
+            @Override
+            public void onResponse(Call<PostResponse> call, Response<PostResponse> response) {
+                int statusCode = response.code();
+                if (statusCode == 201) {
+
```

Comment:
I am not sure about this, but is there no action that we need to call to the view after the creation 201 action ?
We can talk regarding this.